src/shared/components/cancerSummary/SummaryBarGraph.tsx

```diff
@@ -133,21 +136,21 @@ export default class SummaryBarGraph extends React.Component<ISummaryBarGraphPro
     }
 
     private getLegendNames(id:string) {
```

Comment:
`getLegendName(alterationType:keyof ICancerTypeAlterationPlotData)` (not sure if that typing is possible, but the fn and param name should be changed either way)

Reply:
yes